core/executor_pool/tenant_setting.ex

```diff
@@ -126,7 +126,7 @@ defmodule AntikytheraCore.ExecutorPool.TenantSetting do
     :ok
   end
 
-  # To be used by sazabi
+  # To be used by administrative gear
```

Comment:
Sorry for my suggestion in Slack was wrong.

```suggestion
  # To be used by administrative gears
```

Reply:
Thanks!